js/speakers.js

```diff
@@ -0,0 +1,67 @@
+    careerDetails: 'Lorem ipsum dolor sit amet consectetur,adipisicing elit. Repellat aliquam consectetur expedita esse error.',
+  },
+  {
+    guestImage: 'images/image-1.png',
+    guestName: 'Wabosha',
+    guestProfession: 'Director at Nvidia Corp',
+    careerDetails: 'Lorem ipsum dolor sit amet consectetur,adipisicing elit. Repellat aliquam consectetur expedita esse error.',
+  },
+  {
+    guestImage: 'images/image-1.png',
+    guestName: 'Congoleess',
+    guestProfession: 'Director at Nvidia Corp',
+    careerDetails: 'Lorem ipsum dolor sit amet consectetur,adipisicing elit. Repellat aliquam consectetur expedita esse error.',
+  },
+  {
+    guestImage: 'images/image-1.png',
+    guestName: 'Wabosha',
+    guestProfession: 'Director at Nvidia Corp',
+    careerDetails: 'Lorem ipsum dolor sit amet consectetur,adipisicing elit. Repellat aliquam consectetur expedita esse error.',
+  },
+];
+
+function displayInfo() {
+  // section.classList.add('show');
+  document.getElementById('guest1').src = speakerDetails[0].guestImage;
+  document.getElementById('guest-name1').innerHTML = speakerDetails[0].guestName;
+  document.getElementById('guest-prof1').innerHTML = speakerDetails[0].guestProfession;
+  document.getElementById('guest-career1').innerHTML = speakerDetails[0].careerDetails;
+  document.getElementById('guest2').src = speakerDetails[1].guestImage;
+  document.getElementById('guest-name2').innerHTML = speakerDetails[1].guestName;
+  document.getElementById('guest-prof2').innerHTML = speakerDetails[1].guestProfession;
+  document.getElementById('guest-career2').innerHTML = speakerDetails[1].careerDetails;
+  document.getElementById('guest3').src = speakerDetails[2].guestImage;
+  document.getElementById('guest-name3').innerHTML = speakerDetails[2].guestName;
+  document.getElementById('guest-prof3').innerHTML = speakerDetails[2].guestProfession;
+  document.getElementById('guest-career3').innerHTML = speakerDetails[2].careerDetails;
+  document.getElementById('guest4').src = speakerDetails[3].guestImage;
+  document.getElementById('guest-name4').innerHTML = speakerDetails[3].guestName;
+  document.getElementById('guest-prof4').innerHTML = speakerDetails[3].guestProfession;
+  document.getElementById('guest-career4').innerHTML = speakerDetails[3].careerDetails;
+  document.getElementById('guest5').src = speakerDetails[4].guestImage;
+  document.getElementById('guest-name5').innerHTML = speakerDetails[4].guestName;
+  document.getElementById('guest-prof5').innerHTML = speakerDetails[4].guestProfession;
+  document.getElementById('guest-career5').innerHTML = speakerDetails[4].careerDetails;
+  document.getElementById('guest6').src = speakerDetails[5].guestImage;
+  document.getElementById('guest-name6').innerHTML = speakerDetails[5].guestName;
+  document.getElementById('guest-prof6').innerHTML = speakerDetails[5].guestProfession;
+  document.getElementById('guest-career6').innerHTML = speakerDetails[5].careerDetails;
+}
```

Comment:
- [ ] I think we're missing an opportunity here. Since we have an array of speakers with their details. We could iterate through it to generate an HTML node for each one of them. Then we can either append each node to the document or save them all in an array and append them at the end of the iterations.

Use the window.onload event to make sure that these events happened after the whole HTML document has loaded in the browser.

> There is a function that iterates through the speakers list on window load and creates HTML elements with the data from each object. For example:

 ```
window.onload = () => {
  speakerList.map(speaker => {
    // code to build the new HTML element, it can use `createElement()` or just template strings(``).
    // it could attach the new element to a parent inside within the loop
    // or accumulate the generate elements in a variable and attach them all after it
  })
}
```

Reply:
Thank you for this observation. I had however tried to use the for each function. I realize there are better ways to implement but I am having difficulties grabbing the concept. I was hoping this would be optional so that I can work on it on the weekend. Since it works perfectly as per the requirements.